libs/designsystem/header/src/header.component.scss

```
@@ -0,0 +1,49 @@
+@use 'sass:map';
+@use '@kirbydesign/core/src/scss/utils';
+
+.container {
+  width: 100%;
+  padding: utils.size('s') utils.size('s') utils.size('l');
+  display: flex;
+  flex-direction: column;
+
+  @media screen and (min-width: map.get(utils.$breakpoints, 'medium')) {
```

Consider simplifying and aligning with rest of code base:
```suggestion
  @include utils.media('>=medium') {
```